homeassistant/components/hassio/services.yaml

```
@@ -1,37 +1,84 @@
 addon_install:
   description: Install a HassIO docker addon.
   fields:
-    addon: {description: Name of addon., example: smb_config}
-    version: {description: Optional or it will be use the latest version., example: '0.2'}
+    addon:
+      description: Name of addon.
+      example: smb_config
+    version:
+      description: Optional or it will be use the latest version.
+      example: "0.2"
+
 addon_start:
   description: Start a HassIO docker addon.
   fields:
-    addon: {description: Name of addon., example: smb_config}
+    addon:
+      description: Name of addon.
+      example: smb_config
+
+addon_restart:
+  description: Restart a HassIO docker addon.
+  fields:
+    addon:
+      description: Name of addon.
+      example: smb_config
+
+addon_stdin:
+  description: Write data to a HassIO docker addon stdin .
+  fields:
+    addon:
+      description: Name of addon.
+      example: smb_config
+
 addon_stop:
   description: Stop a HassIO docker addon.
   fields:
-    addon: {description: Name of addon., example: smb_config}
+    addon:
+      description: Name of addon.
+      example: smb_config
+
 addon_uninstall:
   description: Uninstall a HassIO docker addon.
   fields:
-    addon: {description: Name of addon., example: smb_config}
+    addon:
+      description: Name of addon.
+      example: smb_config
+
 addon_update:
   description: Update a HassIO docker addon.
   fields:
-    addon: {description: Name of addon., example: smb_config}
-    version: {description: Optional or it will be use the latest version., example: '0.2'}
+    addon:
+      description: Name of addon.
+      example: smb_config
+    version:
+      description: Optional or it will be use the latest version.
+      example: "0.2"
+
 homeassistant_update:
   description: Update HomeAssistant docker image.
   fields:
-    version: {description: Optional or it will be use the latest version., example: 0.40.1}
-host_reboot: {description: Reboot host computer.}
-host_shutdown: {description: Poweroff host computer.}
+    version:
+      description: Optional or it will be use the latest version.
+      example: 0.40.1
+
+host_reboot:
+  description: Reboot host computer.
```

```suggestion
  description: Reboot the host system.
```

Let's avoid "computer" but use `system` instead. (need to be adjusted in other occurrences as well).